strongbox-web-core/src/test/java/org/carlspring/strongbox/config/IntegrationTest.java

```diff
@@ -19,6 +19,7 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.restassured.config.ObjectMapperConfig;
+import io.restassured.module.mockmvc.config.MockMvcParamConfig;
```

Comment:
Is this a non-change?

Reply:
Yeah, it looks like this import slipped into the commit.